scr/community.yaml

```diff
@@ -1,31 +1,19 @@
-name: "SCR Community"
+name: "SCR community"
 source: https://github.com/Sabayon/community-repositories
 directory: /community
-image: sabayon/builder-amd64
+image: sabayon/builder
 namespace: community
 tag_namespace: community
 artefact_path: artifacts
 cache_image: "yes"
 script:
-  - source /etc/profile
-  - enman add https://dispatcher.sabayon.org/sbi/namespace/devel/devel
-  - equo update devel
-  - wget https://raw.githubusercontent.com/Sabayon/devkit/develop/builder -O /usr/sbin/builder
-  - wget https://raw.githubusercontent.com/Sabayon/sabayon-sark/develop/sark-functions.sh -O /sbin/sark-functions.sh
-  - wget https://raw.githubusercontent.com/Sabayon/sabayon-sark/develop/sark-localbuild -O ./sark-localbuild
-  - wget https://raw.githubusercontent.com/Sabayon/sabayon-sark/master/sark-localcreaterepo -O /usr/sbin/sark-localcreaterepo
-  - chmod 0755 /sbin/sark-functions.sh
-  - chmod 0755 /usr/sbin/builder
-  - chmod 0755 ./sark-localbuild
-  - chmod a+x /usr/sbin/sark-localcreaterepo
-  - cp -rfv /sbin/sark-functions.sh /etc/sark-functions
-  - cp -rfv /sbin/sark-functions.sh ./sark-functions.sh
-  - ./sark-localbuild
-environment:
-  - "ACCEPT_LICENSE=*"
-  - "ETP_NONINTERACTIVE=1"
+- sark-localbuild
+- eval 'rm artifacts/build_*.log || true'
+
 storage: "2154014966644308953"
 storage_path: confs
-type: docker
+
+type: docker_execute
```

Comment:
it's better to use `docker` type, `docker_execute` will die :)

Reply:
relics everywhere :)